portfolio/src/main/java/com/google/sps/servlets/ImageUploadFormHandlerServlet.java

```
@@ -64,7 +64,7 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
     List<BlobKey> blobKeys = blobs.get("image");
 
     if (blobKeys == null || blobKeys.isEmpty()) {
-      System.err.println("Key/s are not present");
+      System.err.println("Please upload an image");
```

I guess I missed this change on the previous review. I'd prefer you to `throw` an error rather than print one. That means no returning empty strings. 

This possibly should be in a different PR.

---

ah ok. I'll create those throws then!